markdown/README

```diff
@@ -0,0 +1,5 @@
+Markdown
+========
+
+The Markdown addon parses user input for new items and comments via the Markdown parser.
+This enables users to use the Markdown syntax additionally to the BBCode syntax.
```

Comment:
I like the irony. Adding a Markdown addon, but not using Markdown for the `README.md` but instead plain text :-D

Reply:
That's the result of "copy and paste".